src/index.html

```diff
@@ -1,13 +1,62 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <main class="main">
+      <div class="card" data-qa="card">
+        <img
+        class="card__img"
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27"
+        >
+
+        <div class="card__info">
+          <p class="card__title">
```

Comment:
```suggestion
          <h3 class="card__title">
```
You can use h3 or h4 instead, as it is a title